FrontEnd/myapp/src/App.js

```diff
@@ -1,157 +1,18 @@
-import { Routes, Route, Link } from "react-router-dom";
 import "bootstrap/dist/css/bootstrap.min.css";
-//import Button from "react-bootstrap/Button"
 import "./App.css";
 import "./Table.js";
-
-//import '../node_modules/react-vis/dist/style.css';
-//import {XYPlot, LineSeries} from 'react-vis';
-
-import { Buy, Sell } from "./buysell.js";
+import "./index.css";
+import Navbar from "./Navigation/Navbar.js";
+import "bootstrap/dist/css/bootstrap.min.css";
 
 import React from "react";
-import { useState } from "react";
-
-import { LogIn } from "./Login.js";
-import { SignUp } from "./SignUp.js";
-import { Watchlist } from "./WatchList.js";
-import { StockCheck } from "./StockCheck.js";
-import { Portfolio } from "./Portfolio.js";
-import { useCookies } from "react-cookie";
-import { Dynamic } from "./Dynamic.js";
 
-import { useEffect } from "react";
-import { componentDidMount } from "react";
-import axios from "axios";
 require("dotenv").config();
 
 function App() {
```

Comment:
I like how we now have a clean App.js, makes the base running easier and more class based